config-templates/shopify/customer.yaml

```diff
@@ -0,0 +1,27 @@
+data_source:
+  name: shopify_customers
```

Comment:
maybe make these singular to match the file name? if not, then be consistent with plural / singular through all datasources.